### PR TITLE
[DE] Part 2 for 12.0

### PR DESCRIPTION
**Pull request by Red-Z**
# **SCP:SL Translation PR Template**

## **Language Name**
German

## **State Type of change (New language, changed existing Etc.)**
Added new 12.0 translations for 12.0 and removed unusued files as per PR #1230 and #1221

# **Checklist:**
- [x] I have tested the translation changes ingame
- [x] I have made sure to not commit changes to multiple translations at the same time, or non-translation repo files
- [x] I have ran `EncodingNormalizer.exe` on my changed files before commiting 
   -  *( Exception to this is when edits has __only__ been made in-browser. If this is the case, also* ***check this box*** *)*
- [x] I have made sure that I'm **not** submitting a duplicate translation 
- [x] I have made sure my addition follows guidelines described in the [CONTRIBUTING.md](https://github.com/northwood-studios/SCPSL-Translations/blob/master/.github/CONTRIBUTING.md) file
- [x] I have made sure to label my translation folder by [Microsoft Language Tag Standards](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/a9eac961-e77d-41a6-90a5-ce1a8b0cdb9c)
   - Only in cases of **new** translations (Can be tested [Here](https://rextester.com/WDXPS97501)). Also check this box if the change is **not** a new translation
> For more information regarding game translations, please visit our [README](https://github.com/northwood-studios/SCPSL-Translations/blob/master/README.md) page.

### CODE REVIEW

File: Translations/de/ActionCategory.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Gameplay
+Waffen
+Bewegung
+Schnellzugriff
+Kommunikation
+System
+Nicht zuordenbar
```
Comment (Dat-Joseph) [inline Translations/de/ActionCategory.txt:7]: "Nicht zuweisbar" ?

File: Translations/de/DeathReasons.txt [hunk 2]
```diff
@@ -29,4 +29,5 @@ Anzeichen eines schweren Traumas an beiden Oberschenkelknochen deuten auf einen
 Du wurdest geopfert um SCP-106 wieder einzudämmen.
 Automatisch für Teambeschuss getötet.
 Kalte Haut und erfrorene Finger deuten auf einen Tod durch Hyperthermie hin.
-Du bist erfroren.
\ No newline at end of file
+Du bist erfroren.
+Der Körper weist tiefe Risswunden und Kratzer auf.
```
Comment (Dat-Joseph) [inline Translations/de/DeathReasons.txt:33]: "Platzwunden" might be better than "Risswunden".

File: Translations/de/HumanAbilities.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Sprinten:Erhöht deine Bewegungsgeschwindigkeit, aber auch die deine Laufgeräusche.
+Schleichen:Deine Bewegungen werden leiser auf Kosten der Geschwindigkeit.
+Inventar:Öffne dein Inventar und wähle einen Gegenstand mit Links-Klick aus, oder lasse es mit Rechts-Klick fallen.
+Unbewaffnet:Du hast gar nichts. Finde etwas, egal was.
+Zugangskarte:Du hast eine Zugangskarte, nicht die beste, aber immerhin. Verlier sie nicht!
+Standard Ausrüstung:Du bist mit einer Waffe, einer Zugangskarte und einigen weiteren nützlichen Gegenständen ausgerüstet.
+Bewaffnet & Gefährlich:Sie wurden mit schwerer Bewaffnung, einer Zugangskarte und medizinischen Gegenständen ausgerüstet.
```
Comment (Dat-Joseph) [inline Translations/de/HumanAbilities.txt:7]: It's not a good thing to first refer to someone informal and then formal again. 
If we go with "Du", then we shouldn't use "Sie" in other situations. I suggest we stick by "Du" as it relays a sense of friends or close to another. Makes it more welcome for new players.

File: Translations/de/SCP049_HUD.txt [hunk 4]
```diff
@@ -11,3 +11,11 @@ Drücke [key], um eine weitere Wiederbelebung zu verweigern.
 Kein Ziel gefunden.
 Du bist bereits bei voller Gesundheit.
 Diese Leiche wird bereits aufgefressen.
+Langsam und zerbrechlich, SCP-049 spezialisiert sich in hinterhältige Angriffe. Fähig das Blatt des Spieles, mit Horden von "geheiltem" Personal, zu wenden.
+Schnell und gefährlich, SCP-049-2 Instanzen schwärmen aus, um Panik und Chaos für Ihren Meister zu verursachen, so können neue Körper dem Spiel beitreten.
+Herzinfarkt: Du streckst deine behandschuhte Hand aus, um die Herzen derjenigen zu berühren, die an der Pestilenz leiden, wodurch ihre Körperfunktionen schnell versagen. Das gibt dir die Möglichkeit, deine Heilung durchzuführen.
```
Comment (Dat-Joseph) [inline Translations/de/SCP049_HUD.txt:16]: I think "anzuwenden" might be better than "durchzuführen".

File: Translations/de/SCP049_HUD.txt [hunk 5]
```diff
@@ -11,3 +11,11 @@ Drücke [key], um eine weitere Wiederbelebung zu verweigern.
...
+Der gute Sinn des Doktors:Du konzentrierst dich voll und ganz auf eine verdorbene Seele. Du bewegst dich schneller, entschlossen, sie von der Seuche zu befreien. Du bist in der Lage, die Entfernung eines Opfers anhand des üblen Geruchs der Seuche zu bestimmen.
```
Comment (Dat-Joseph) [inline Translations/de/SCP049_HUD.txt:17]: Du konzentrierst dich voll und ganz auf eine verdorbene Seele. Du bewegst dich schneller _und entschlossen auf sie zu, um sie von der Seuche zu befreien._ Du bist in der Lage, die Entfernung eines Opfers anhand des üblen Geruchs der Seuche zu bestimmen.

The underlined text might be better.

File: Translations/de/SCP079_HUD.txt [hunk 6]
```diff
@@ -65,3 +65,10 @@ ZIELE
 SCP SUBJEKTE
 ZIVILIST
 MILITÄR
+LAUTSPRECHER VERWENDEN
+AUFZUG ZUGRIFF VERWEIGERT
+SCP-079 hat keine direkten Kampf- oder Bewegungsmöglichkeiten. Stattdessen sehen Sie die Welt durch Sicherheitskameras, die in jedem Raum zu finden sind, und interagieren mit diesen. Alle verfügbaren Fähigkeiten werden in der oberen rechten Ecke Ihres HUDs aufgelistet.
```
[the remark this answers is not in the document]
Comment (Dat-Joseph) [reply Translations/de/SCP079_HUD.txt:70]: Again, the "Sie" and "Du" thing

File: Translations/de/SCP106_HUD.txt [hunk 7]
```diff
@@ -1,3 +1,8 @@
 ANGRIFF COOLDOWN
 Nicht verfügbar auf der Surface.
 Nicht verfügbar während Aufzugsfahrt.
+Eine unaufhaltsame Kraft, die in der Lage ist, jede Tür in der Einrichtung problemlos zu überwinden.
+Gefangennahme:Fängt einen Menschen aus nächster Nähe und sperrt ihn in die Pocket Dimension ein - ein tödliches Gefängnis, das durch deine Regeln bestimmt wird. Die Freude am Fangen neuer Beute füllt die Vitalität wieder auf.
+Stalk:Taucht dich in den Boden ein. Bewegen dich schnell und unsichtbar, regeneriere dich und bereiten dich darauf vor, deine Beute anzugreifen. Benötigt Vitalität, die sich langsam regeneriert, während Stalk inaktiv ist.
```
Comment (Dat-Joseph) [inline Translations/de/SCP106_HUD.txt:6]: A few letters to much in your words here and there. 

"Stalk:Taucht dich in den Boden ein. Beweg dich schnell und unsichtbar, regeneriere dich und bereite dich darauf vor, deine Beute anzugreifen. Benötigt Vitalität, die sich langsam regeneriert, während Stalk inaktiv ist."

File: Translations/de/SCP939_HUD.txt [hunk 8]
```diff
@@ -10,3 +10,11 @@ Löschen
 OK
 Mehr Informationen
 SCP-939 verwendet eine Mechanik, wobei die letzten wenigen Sekunden deines Sprachchats temporär aufgenommen werden. Dies kann durch SCP-939 wieder abgespielt werden, um Ziele anzulocken.\nDie Aufnahmen werden automatisch gelöscht, wenn du den Server verlässt, SCP-939 stirbt, oder die Runde endet. Wenn du dieses Feature ausschaltest, werden alle deine existierenden Aufnahmen gelöscht.
+Die Wolke kann an dieser Stelle nicht eingesetzt werden.
+Die minimale Wolken-Größe wurde nicht erreicht. Halten Sie die Taste länger gedrückt.
```
Comment (Dat-Joseph) [inline Translations/de/SCP939_HUD.txt:14]: "Halte die Taste länger gedrückt."
"Sie" and "Du" again.

File: Translations/de/SCP939_HUD.txt [hunk 9]
```diff
@@ -10,3 +10,11 @@ Löschen
...
+SCP-939 ist blind und sieht durch Schall. Es nutzt Fehler aus, indem es mit lautlosen Schritten und beängstigender Geschwindigkeit Hinterhalte vorbereitet und zum Töten aus dem Nichts auftaucht.
+Krallen:Klicke, um deine Klaue in einem weiten Bogen vor dir zu schwingen und deinem Opfer Fleisch und Knochen zu entreißen. Während du dich fokussierst, kannst du dich mit deinem primären Angriff auf deine Beute stürzen.
+Fokus:Wenn du deine gesamte Energie auf deine Sinne richtest, kannst du deine Beute aus größerer Entfernung sehen und anhand ihrer Aktionen erkennen, um welche Art von Beute es sich handelt.
+Amnestische Wolke:Fülle deine Lunge mit einem gefährlichen neurochemischen Gas, das in einem Radius ausgestoßen wird, der durch die Zeit bestimmt wird, in der es durch Halten der Taste vergrößert wurde. Menschen, die das Gas einatmen, können  SCP-939 nicht mehr wahrnehmen und bestimmte Aktionen nicht mehr ausführen. Aber Vorsicht:Wenn du angreifst oder angegriffen wirst, wirst du für eine kurze Zeit sichtbar.
```
Comment (Dat-Joseph) [inline Translations/de/SCP939_HUD.txt:18]: Seemingly a double spacebar between "können" and "SCP-939". Could be that GitHub is bugged on my end though.

File: Translations/de/Voicechat.txt [hunk 10]
```diff
@@ -17,3 +17,7 @@ Bestimmten Spielmechaniken erlauben, kurze Aufnahmen deiner Stimme zu verwenden.
 Einstellungen übernehmen
 Zurück
 Klicke "[key]" und navigiere zu "Einstellungen" zum Ändern.
+Lebende Spieler stummschalten
+Andere Zuschauer stummschalten
+Lobby stummschalten
+Globalen Sprachchat bei Rundenzusammenfassung stummschalten
```
Comment (Dat-Joseph) [inline Translations/de/Voicechat.txt:23]: We can test this, if "Rundenzusammenfassung" is too long, we can replace it with "Rundenende" later. Else it's fine.

**Comment by Dat-Joseph** [comment]
Review done for now, change the marked sections or add suggestions to it if you want.

**Comment by Red-Z** [comment]
@Dat-Joseph updated with translations of #1238 cause this PR wasn't merged yet

**Comment by Red-Z** [comment]
Updated again reflecting changes in #1259

**Pull request merged**